src/app/components/audio-recordings/pages/download/download.component.html

```diff
@@ -328,9 +328,9 @@ <h3>5. Run the script</h3>
     </span>
   </p>
 
-  <code id="guest-run-script" *ngIf="!session.authToken">{{
-    runScriptCommand
-  }}</code>
+  <div id="guest-run-script" class="alert alert-warning" *ngIf="!session.authToken">
+    <i>You will have to log in to see this command.</i>
+  </div>
```

Comment:
add an anchor to the sign in view, and make sure it redirects back to this view after a user clicks on it